src/TouristInfoCards.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import card from "./data/card";
+
+export default function TouristInfoCards() {
+  const cardInfo = card();
```

Comment:
this works, but you could also just have your data in your `card` file saved a variable and export it, rather than needing to call a function to get your data